virt/virt-4-10-release-notes.adoc

```diff
@@ -196,44 +194,69 @@ $ oc annotate --overwrite -n openshift-cnv hyperconverged kubevirt-hyperconverge
 +
-[source,yaml]
+[source,terminal]
 ----
-apiVersion: kubevirt.io/v1alpha3
-kind: VirtualMachine
-metadata:
-  annotations:
-...
+$ oc annotate --overwrite -n openshift-cnv hco kubevirt-hyperconverged 'networkaddonsconfigs.kubevirt.io/jsonpatch=[
+  {
+    "op": "replace"
+    "path": "/spec/kubeMacPool"
+    "value": null
+  }
+ ]'
 ----
-+
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1979114[*BZ#1979114*])
-+
-** When connecting to the VNC console by using the {VirtProductName} web console, the VNC console always fails to respond.
-+
-*** As a workaround, create the virtual machine from the CLI or upgrade to {VirtProductName} 4.8.
-+
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1977037[*BZ#1977037*])
 
-//Known issues discovered after 4.9 release
+* {VirtProductName} links a service account token in use by a pod to that specific pod. {VirtProductName} implements a service account volume by creating a disk image that contains a token. If you migrate a VM, then the service account volume becomes invalid. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2037611[*BZ#2037611*])
+** As a workaround, use user accounts rather than service accounts because user account tokens are not bound to a specific pod.
+
+* {VirtProductName} does not process multiple attempts to stop a VM. If a VM crashes during shutdown, then you cannot issue a new stop attempt using `--force --grace-period 0` flags and you cannot easily remove the VM from the UI. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2040766[*BZ#2040766*])
+** As a workaround, delete the virt-launcher pod directly.
+
+* If you configure the `HyperConverged` custom resource (CR) to enable mediated devices before drivers are installed, enablement of mediated devices does not occur. This issue can be triggered by updates. For example, if `virt-handler` is updated before `daemonset`, which installs NVIDIA drivers, then nodes cannot provide virtual machine GPUs. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2046298[*BZ#2046298*])
+** As a workaround:
+. Remove `mediatedDevicesConfiguration` and `permittedHostDevices` from the `HyperConverged` CR.
+. Update both `mediatedDevicesConfiguration` and `permittedHostDevices` stanzas with the configuration you want to use.
 
-* If a single node contains more than 50 images, pod scheduling might be imbalanced across nodes. This is because the list of images on a node is shortened to 50 by default.
-** As a workaround, you can disable the image limit by xref:../nodes/nodes/nodes-nodes-managing.adoc#nodes-nodes-managing[editing the `KubeletConfig` object] and setting the value of `nodeStatusMaxImages` to `-1`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1984442[*BZ#1984442*])
+* YAML examples in the VM wizard are not always up to date with the latest information because the examples are hardcoded upstream. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2055492[*BZ#2055492*])
```

Comment:
The examples aren't hardcoded upstream; they might change upstream, and the hardcoded examples in the VM wizard won't change with them. I'd rephrase the sentence to:
```
* YAML examples in the VM wizard are hardcoded and do not always contain the latest upstream changes.
```